### PR TITLE
Create fibonacci.cpp

### PR DESCRIPTION
**Pull request by PRIYANSHU-AMBITION**
## Have you read the Contribution Guidelines?

yes

## Description

fibonacci program in cpp

## Checklist
- [Y] I've read the contribution guidelines.
- [Y] I've checked the issue list before deciding what to submit.
- [Y] I've edited the README.md and link to my code.

### CODE REVIEW

**Review by mephi007 — CHANGES_REQUESTED** [review]
Can you please modify Readme.md file too.

File: C-Plus-Plus/fibonacci.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ clrscr();
+ cout<<"\n input the no. of terms for fibonacci series";
+ cin>>n;
+ while(i<n)
+ {
+  cout<<" "<<fibonacci(i);
+  i++;
+ }
+ return 0;
+}
```
Comment (mephi007) [inline C-Plus-Plus/fibonacci.cpp:27]: Add sample Input and Output, Time and Space Complexity as comment.

**Review by mephi007 — CHANGES_REQUESTED** [review]

**Comment by sukritishah15** [comment]
@PRIYANSHU-AMBITION - Please make the requested changes.
Thank you!

**Comment by sukritishah15** [comment]
You can create a new PR, but, we are closing this for now.
Thank you!

**Pull request closed without merging**